docs/_data/components/schemas/forecast-limit-period.yaml

```diff
@@ -1,16 +1,43 @@
-slim:
+forecast-period-proposal:
   type: object
+  title: Forecasted Period
+  description: |
+
+    A Forecasted Period contains proposed limits for a power system
+    object, e.g., a segment, that were forecasted for a particular Period
+    in a Forecast Window. The status of the proposal as determined by the
+    Clearinghouse is also included.
+
   properties:
     period-start:
       $ref: ./period-start.yaml
-    updated-time:
-      $ref: ./timestamp.yaml
-    values:
-      $ref: ./limit-value-set.yaml
+    period-end:
+      $ref: ./period-start.yaml
+    continuous-operating-limit:
+      $ref: ./limit.yaml
+    emergency-operating-limits:
+      $ref: ./array-max-emergency-durations.yaml#/limit-value-set
+    status:
+      $ref: ./proposal-status.yaml
```

Comment:
@getorymckeag Is this needed for interop? I can imagine it being *useful*, maybe even just at the top-level forecast proposal instead of the individual periods, but I don't know if it is *necessary*. If it is necessary--that is, its not case of YAGNI for interop--I'd bet we could define an enum for it. As is, I'd have questions about how to understand it, since the cleared facility continuous operating limit could be from one proposal with the emergency operating limits from a different provider.

Reply:
Agreed, remove.